test/integration/apiserver/apply/resetFields_test.go

```diff
@@ -130,59 +129,19 @@ func TestApplyResetFields(t *testing.T) {
 					t.Fatal(err)
 				}
 
-				True := true
-				obj, err := dynamicClient.
+				// an unforced apply should not conflict if the resetfields are set correctly
```

Comment:
please correct me on this.
I changed it, because Certs made some trouble.
As they don't own anything on create, my test failed with "create manager not found".
I played around with managedFields decoding and doing intersects etc. but nothing really worked. Then i thought about this and so far it looks good.
Is this a reasonable assumption?

Reply:
Thanks for writing this test. Why would it conflict in the first place if you're applying the same status?

Reply:
Because I'm applying with a different fieldManager

Reply:
Right, that should work